src/deprecated.jl

```diff
@@ -1,7 +1,9 @@
-using Base: @deprecate, @deprecate_binding
+using Base: @deprecate
 
-# BEGIN TimeZones 0.9 deprecations
+# BEGIN TimeZones 1.0 deprecations
```

Comment:
1.0, huh

Reply:
It's time. I usually don't make API breaking changes but I use minor versions to indicate features.